Explorer/Assets/DCL/AvatarRendering/Wearables/Components/IWearable.cs

```diff
@@ -21,14 +23,73 @@ public interface IWearable : IAvatarAttachment
         /// </summary>
         bool TryResolveDTO(StreamableLoadingResult<WearableDTO> result);
 
-        void ResolvedFailedDTO(StreamableLoadingResult<WearableDTO> result);
-
         bool TryGetFileHashConditional(BodyShape bodyShape, Func<string, bool> contentMatch, out string? hash);
 
         void GetHidingList(string bodyShapeType, HashSet<string> hideListResult);
 
         bool IsCompatibleWithBodyShape(string bodyShape);
 
         bool HasSameModelsForAllGenders();
+
+        class Fake : IWearable
+        {
+            private readonly string? mainHash;
+            private HashSet<string> expectedUpperWearableHide;
+
+            public bool IsLoading { get; }
+
+            public void UpdateLoadingStatus(bool isLoading)
+            {
+                //ignore
+            }
+
+            public StreamableLoadingResult<SceneAssetBundleManifest>? ManifestResult { get; set; }
+            public StreamableLoadingResult<Sprite>? ThumbnailAssetResult { get; set; }
+            public AvatarAttachmentDTO DTO { get; }
+            public StreamableLoadingResult<WearableDTO> Model { get; set; }
+
+            public WearableType Type { get; }
+
+            public WearableAssets[] WearableAssetResults { get; } = Array.Empty<WearableAssets>();
```

Comment:
You already initialize this property from the ctor. And I wonder if we really need a separate fake or we can create a Substitute from a static method?